src/python_testing/TC_WEBRTCP_2_17.py

```diff
@@ -0,0 +1,165 @@
+            node_id=self.dut_node_id, fabric_index=self.default_controller.GetFabricIndexInternal(), endpoint=endpoint
+        )
+
+        self.step(2)
+        # Send ProvideOffer command with null WebRTCSessionID and valid SDP offer
+        logger.info("Sending ProvideOffer command with null WebRTCSessionID")
+
+        # Create a valid SDP offer
+        webrtc_peer.create_offer()
+        offer = await webrtc_peer.get_local_offer()
+
+        resp: Clusters.WebRTCTransportProvider.Commands.ProvideOfferResponse = await webrtc_peer.send_command(
+            cmd=Clusters.WebRTCTransportProvider.Commands.ProvideOffer(
+                webRTCSessionID=NullValue,
+                sdp=offer,
+                streamUsage=Clusters.Objects.Globals.Enums.StreamUsageEnum.kLiveView,
+                videoStreamID=videoStreamID,
+                audioStreamID=audioStreamID,
+                originatingEndpointID=1,
+            ),
+            endpoint=endpoint,
+            payloadCapability=ChipDeviceCtrl.TransportPayloadCapability.LARGE_PAYLOAD,
+        )
+
+        asserts.assert_equal(type(resp), Clusters.WebRTCTransportProvider.Commands.ProvideOfferResponse,
+                             "Incorrect response type")
+        session_id = resp.webRTCSessionID
+        asserts.assert_true(session_id >= 0, f"Invalid session ID: {session_id}")
+        logger.info(f"DUT allocated WebRTC session ID: {session_id}")
+        webrtc_manager.session_id_created(session_id, self.dut_node_id)
+
+        self.step(3)
+        # Wait for incoming Answer command from DUT on WebRTC Requestor cluster
+        logger.info(f"Waiting for Answer command from DUT for session {session_id}")
+
+        # Wait for the Answer command from the DUT
+        answer_sessionId, answer_sdp = await webrtc_peer.get_remote_answer()
+
+        # Verify the Answer command contains the same session ID
+        asserts.assert_equal(answer_sessionId, session_id,
+                             f"Answer session ID {answer_sessionId} does not match expected {session_id}")
+        logger.info(f"Received Answer command for session {session_id}")
+
+        self.step(4)
+        # Verify the Answer command contains valid SDP answer content
+        logger.info("Verifying SDP answer content")
+
+        # Basic validation of SDP answer format
+        asserts.assert_true(len(answer_sdp) > 0, "SDP answer is empty")
+        asserts.assert_equal(type(answer_sdp), str, "SDP answer has incorrect type")
+
+        # Verify SDP answer contains required session-level fields
+        asserts.assert_true("v=0" in answer_sdp, "SDP answer missing version line (v=0)")
+        asserts.assert_true("s=" in answer_sdp, "SDP answer missing session name line (s=)")
+        asserts.assert_true("t=" in answer_sdp, "SDP answer missing timing line (t=)")
+
+        # Verify SDP answer contains media descriptions
+        asserts.assert_true("m=" in answer_sdp, "SDP answer missing media description line (m=)")
```

Comment:
Shouldn't we be explicitly testing for two m= lines,  one that is video, one that is audio?

Reply:
```suggestion
        asserts.assert_true("m=" in answer_sdp, "SDP answer missing media description line (m=)")
```

Reply:
acked